hack/run-tests.sh

```diff
@@ -310,6 +276,8 @@ case $1 in
 esac
 done
 
+source $SCRIPT_DIR/crds-utils.sh
```

Comment:
This can be put inside the `if ${CRDS_UPDATE}; then`

Reply:
Changed